ui/app/components/secret-engine/configure-wif.hbs

```diff
@@ -3,64 +3,94 @@
-  <MessageError @errorMessage={{this.errorMessage}} />
+  {{! AWS specific note and section header }}
+  {{#if (eq @type "aws")}}
+    <p class="has-text-grey-dark has-top-bottom-margin">
+      Note: the client uses the official AWS SDK and will use the specified credentials, environment credentials, shared file
+      credentials, or IAM role/ECS task credentials in that order.
+    </p>
+    <h2 class="title is-5 has-bottom-padding-s has-top-margin-l" data-test-access-title>
+      Access to AWS
+    </h2>
+  {{/if}}
   <div class="box is-fullwidth is-sideless">
-    {{! accessType can be "azure" or "wif" - since WIF is an enterprise only feature we default to "azure" for community users and only display those related form fields. }}
+    {{! Only enterprise users can change access type from "account" to "wif" }}
     {{#if this.version.isEnterprise}}
       <fieldset class="field form-fieldset" id="protection" data-test-access-type-section>
         <legend class="is-label">Access Type</legend>
         <p class="sub-text" data-test-access-type-subtext>
           {{#if this.disableAccessType}}
             You cannot edit Access Type if you have already saved access credentials.
           {{else}}
-            Choose the way to configure access to Azure. Access can be configured either using an Azure account or with the
-            Plugin Workload Identity Federation (WIF).
+            Choose the way to configure access to
+            {{@displayName}}. Access can be configured either using
+            {{if (eq @type "aws") "IAM access keys" (concat @displayName " account credentials")}}
+            or with the Plugin Workload Identity Federation (WIF).
           {{/if}}
         </p>
         <div>
           <RadioButton
-            id="access-type-azure"
-            name="azure"
+            id="access-type-{{@type}}"
+            name="account"
             class="radio"
-            data-test-access-type="azure"
-            @value="azure"
+            data-test-access-type={{@type}}
+            @value="account"
             @groupValue={{this.accessType}}
-            @onChange={{fn this.onChangeAccessType "azure"}}
+            @onChange={{this.onChangeAccessType}}
             @disabled={{this.disableAccessType}}
           />
-          <label for="access-type-azure">Azure account credentials</label>
+          <label for="access-type-{{@type}}">{{@displayName}} account credentials</label>
           <RadioButton
             id="access-type-wif"
             name="wif"
             class="radio has-left-margin-m"
             data-test-access-type="wif"
             @value="wif"
             @groupValue={{this.accessType}}
-            @onChange={{fn this.onChangeAccessType "wif"}}
+            @onChange={{this.onChangeAccessType}}
             @disabled={{this.disableAccessType}}
           />
           <label for="access-type-wif">Workload Identity Federation</label>
         </div>
       </fieldset>
     {{/if}}
```

Comment:
I brought the MessageError component down some. It didn't look amazing that high up on the form. Happy to move back if folks disagree with the placement. Here is how it looks moved. 
![image](https://github.com/user-attachments/assets/8af37239-598a-4bc7-b034-3e68327bfd86)

Reply:
This is probably a good question for design? The user doesn't know it's two separate endpoints so I think halfway through the form seems a little odd

Reply:
I have a question into design, stay tuned.

Reply:
moved it back per designs feedback